camelot/view/action_steps/crud.py

```diff
@@ -108,14 +115,55 @@ class SetColumns(ActionStep):
     
     def __init__(self, static_field_attributes):
         self.static_field_attributes = static_field_attributes
+        self.column_attributes = []
```

Comment:
zouden we hier dan geen lijst [('type', 'attrs')] van maken, zodat self.static_field_attributes weg kan ?